packages/ingest/src/consumer.ts

```diff
@@ -42,9 +44,15 @@ export class Consumer {
   ): Promise<0 | 1> {
     try {
       await fn(this.queueUrl, message);
-      if (this.deleteProcessedMessage) await deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      if (this.deleteProcessedMessage) {
+        await sqs.deleteSQSMessage(this.queueUrl, message.ReceiptHandle);
+      }
       return 1;
     } catch (error) {
+      if (error.code === 'ExecutionAlreadyExists') {
+        log.info('Execution already exists. Proceeding to delete message from queue...');
```

Comment:
This is the message from sqs2sfThrottle:
`"message": "Execution already exists. Proceeding to delete message from queue...",`
It would be useful to log the error as of the sqs2sf lambda, so we know which execution already exists.

And we don't need to log the following error in the case of `ExecutionAlreadyExists`:
```
2023-06-16T01:29:29.546Z	c006a9dd-937e-4020-9c2d-f68af4332fdc	ERROR	{
    "error": {
        "name": "ExecutionAlreadyExists",
        "message": "Execution Already Exists: 'arn:aws:states:us-east-1:596205514787:execution:waitPassTestSf_1686878903201:startSfSpec_12345'",
        "stack": [
            "ExecutionAlreadyExists: Execution Already Exists: 'arn:aws:states:us-east-1:596205514787:execution:waitPassTestSf_1686878903201:startSfSpec_12345'",
            "    at Request.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/json.js:61:27)",
            "    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)",
            "    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:78:10)",
            "    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:686:14)",
            "    at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)",
            "    at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)",
            "    at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10",
            "    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)",
            "    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:688:12)",
            "    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:116:18)"
        ]
    },
    "level": "error",
    "message": "Execution Already Exists: 'arn:aws:states:us-east-1:596205514787:execution:waitPassTestSf_1686878903201:startSfSpec_12345'",
    "sender": "@cumulus/ingest/consumer",
    "timestamp": "2023-06-16T01:29:29.546Z"
}
```

Reply:
https://github.com/nasa/cumulus/pull/3389/commits/1c7a7c0eb17856a6fec53688ed514cc02f1cf58f I updated the logging for sqs2sfThrottle to log.debug the error, and then log.debug the message being deleted. What do you think @jennyhliu ? 

```
{
    "level": "debug",
    "message": "ExecutionAlreadyExists: Execution Already Exists: 'arn:aws:states:us-east-1:xxxx:execution:jtran-tf-IngestGranule:9775ec96-5684-44f5-ac72-0b343c3e8775'\n    at Request.extractError (/var/runtime/node_modules/aws-sdk/lib/protocol/json.js:61:27)\n    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:106:20)\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:78:10)\n    at Request.emit (/var/runtime/node_modules/aws-sdk/lib/request.js:686:14)\n    at Request.transition (/var/runtime/node_modules/aws-sdk/lib/request.js:22:10)\n    at AcceptorStateMachine.runTo (/var/runtime/node_modules/aws-sdk/lib/state_machine.js:14:12)\n    at /var/runtime/node_modules/aws-sdk/lib/state_machine.js:26:10\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:38:9)\n    at Request.<anonymous> (/var/runtime/node_modules/aws-sdk/lib/request.js:688:12)\n    at Request.callListeners (/var/runtime/node_modules/aws-sdk/lib/sequential_executor.js:116:18) {\n  code: 'ExecutionAlreadyExists',\n  time: 2023-06-16T19:20:30.434Z,\n  requestId: '44cee665-9b61-4d5a-88ed-6aa29978f09b',\n  statusCode: 400,\n  retryable: false,\n  retryDelay: 83.27922326267347\n}",
    "sender": "@cumulus/ingest/consumer",
    "timestamp": "2023-06-16T19:20:30.491Z"
}
{
    "level": "debug",
    "message": "Deleting message for execution that already exists...",
    "sender": "@cumulus/ingest/consumer",
    "timestamp": "2023-06-16T19:20:30.491Z"
}
{
    "level": "debug",
    "message": "Completed deleting message.",
    "sender": "@cumulus/ingest/consumer",
    "timestamp": "2023-06-16T19:20:30.561Z"
}
```

Reply:
We do want to log stack trace for the other errors.

Reply:
https://github.com/nasa/cumulus/pull/3389/commits/2c6c58da87893d4bd6300fb2592ae57ced75c940